frontend/packages/core-api-client/src/lib/client/BaseRESTClient.ts

```diff
@@ -64,4 +65,58 @@ export class BaseRESTClient {
     }
     return clientResponse<TRequest, TBody>(value, request, expectStatusCode);
   }
+
+  public async makeRequest<TRequestBody, TResponseBody>(
```

Comment:
does this actually need to be public?

Reply:
No. My thinking was that it may be useful to have access to the generic method, but maybe this can be private for now.